api/src/main/java/org/openmrs/module/metadatadeploy/bundle/CoreConstructors.java

```diff
@@ -56,6 +60,16 @@
 public class CoreConstructors {
 
 	/**
+	 * Constructs a dbsessionFactory source
+	 * @return the transient object
+	 */
+	public static DbSessionFactory dbSessionFactory(SessionFactory sessionFactory) {
```

Comment:
so that is why i was struggling to include DBSessionFactory Constructor in the CoreConstructor...Updating openmrs version to 1.9.9 in the pom.xml had no effect.